stretch_core/nodes/keyboard_teleop

```diff
@@ -1,33 +1,12 @@
 #!/usr/bin/env python
 
-from __future__ import print_function
```

Comment:
I'd like to keep this on all our files to help with a future migration to Python 3.